start-siyuan.sh

```diff
@@ -2,6 +2,25 @@
 
 set -oue pipefail
 
+WORKSPACE_CONFIG_FILE="/home/$USER/.config/siyuan/workspace.json"
+
+# Creating default workspace configuration when first time opening SiYuan
+# then Siyuan will initialize directly.
+#
+# For users who installed SiYuan, please copy folder `~/SiYuan` in host
+# to `~/.var/app/org.b3log.siyuan/SiYuan` to recover your data, but dont
+# forget to clean up `~/.var/app/org.b3log.siyuan/SiYuan` to empty at first.
+if [ ! -s "$WORKSPACE_CONFIG_FILE" ]; then
+    echo "[\"/home/$USER/SiYuan\"]" > "$WORKSPACE_CONFIG_FILE";
```

Comment:
this can break anytime if the home dir is not in `/home`. It doesn't have to be.

Use `$HOME`.

Reply:
Okay, thanks your suggestion. I will consider applying it later.

Reply:
Now I used `$HOME` to locate HomeDir instead of `/home/$USER`.

Thanks for your suggestion again.